Control/DeepSeq.hs

```diff
@@ -105,6 +114,10 @@ import GHC.Stack ( CallStack(..) )
 import GHC.SrcLoc ( SrcLoc(..) )
 #endif
 
+#if __GLASGOW_HASKELL__ >= 708
```

Comment:
Indeed, in general I dislike `__GLASGOW_HASKELL__` being abused as a proxy for `MIN_VERSION_base`.

In this case, however, it's not so clear cut IMHO, as `Coercible` is both a `base`-feature  as well as depending on facilities provided only by certain GHC versions (although I'm not sure if you couldn't actually fake/emulate `Data.Coerce.coerce` via `unsafeCoerce`... in which case I would see this more as a case for `MIN_VERSION_base`)